specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -1033,6 +1033,68 @@
         }
       }
     },
+    "/skillsets('{skillsetName}')/search.resetskills": {
+      "post": {
+        "tags": [
+          "Skillsets"
+        ],
+        "operationId": "Skillsets_ResetSkills",
+        "x-ms-examples": {
+          "SearchServiceResetSkills": {
+            "$ref": "./examples/SearchServiceResetSkills.json"
+          }
+        },
+        "description": "Reset an existing skillset in a search service.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/searchservice/preview-api/reset-skills"
```

Comment:
We should use an aka.ms URL here. It will make it more stable and will be easier to change when the feature goes GA.

Reply:
how do we generate an aka.ms URL?